charts/OpenVPN/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+description: DEPRECATED - A Helm chart to install an openvpn server inside a kubernetes cluster.  Certificate
+  generation is also part of the deployment, and this chart will generate client keys
+  as needed.
+name: openvpn
+version: 4.2.5
+appVersion: 1.1.0
+deprecated: true
```

Comment:
Can you please remove this parameter else it will be shown as a deprecated chart on the UI